roboteam_ai/src/control/PositionUtils.cpp

```diff
@@ -85,7 +85,9 @@ std::vector<Vector2> PositionUtils::getPenaltyPositions(int number) {
                                  {lengthOffset, widthOffset},
                                  {lengthOffset, - widthOffset},
                                  {0, - widthOffset},
-                                 {- lengthOffset, - widthOffset}};
+                                 {- lengthOffset, - widthOffset},
```

Comment:
If I understand correctly this is just adding two positions in case we have a penalty, am I correct?

Reply:
yes :-) This fixed a segfault because with 8 robots there is one without a formation (and 2 if there is no keeper set)